aries_vcx/src/protocols/proof_presentation/verifier/state_machine.rs

```diff
@@ -239,12 +243,28 @@ impl VerifierSM {
                     }
                     Err(err) => {
                         let problem_report = build_problem_report_msg(Some(err.to_string()), &self.thread_id);
+
                         let sm = match err.kind() {
                             AriesVcxErrorKind::InvalidProof => VerifierFullState::Finished(
                                 (state, presentation, PresentationVerificationStatus::Invalid).into(),
                             ),
                             _ => VerifierFullState::Finished((state, problem_report.clone()).into()),
                         };
+
+                        // Use the ProblemReport message as part of
```

Comment:
Given that ProblemReport in Notification family is quite explicitly expressed with new type `NotificationProblemReport` I find this comment to not carry much information. But perhaps including link to
https://github.com/hyperledger/aries-rfcs/tree/main/features/0015-acks#relationship-to-problem-report
instead would be nice here

Reply:
Removed.

The link itself would be more suitable in the type definition itself, for `ack` messages as well. Should I add them as part of this PR?